spec/fixtures/static_pages/.html

```diff
@@ -4,6 +4,8 @@
 <meta charset='utf-8'>
 <meta content='IE=edge' http-equiv='X-UA-Compatible'>
 <meta content='width=device-width, initial-scale=1' name='viewport'>
+<meta content='1234' property='fb:admins'>
```

Comment:
@Taz17 this looks very suspicious. I would guess this is the source of the problem. Why was this committed?

Reply:
Those were changes made automatically after `bundle exec rake` was run. I manually anything under `spec/*`

Reply:
It those get changed, that means the regression tests failed!

Reply:
Yes definitely, that was in commit [42ba8f1](https://github.com/openaustralia/publicwhip/pull/1329/commits/42ba8f1128bc2e5690e2ccfe6ab3f0e77a1d9993)

I then re-ran the rake and it added more changes. Here [33cca66](https://github.com/openaustralia/publicwhip/pull/1329/commits/33cca66676ffbbda81fe3e73e11b5f7561870532)

But its passing everything now?

Reply:
Why is something to do with a meta tag to do with facebook getting updated in a PR to do with searching?

Reply:
I don't know, all the changes that I made are regarding search.

What could be the cause of the meta tag getting updated?

As a side note. I just ran the rake tasks on the branch ([policy_filter_search_js](https://github.com/Taz17/publicwhip/tree/policy_filter_search_js)) to see why PR https://github.com/openaustralia/publicwhip/pull/1332 was failing the rake tasks and I got the same changes for the facebook meta tags... very odd

Reply:
I also get the same meta tag changes if i run `bundle exec rake` on the master branch (note: i never code on the master branch)

Reply:
What you need to do is understand where that meta tag is being generated and then figure out what you're doing to cause that to be different on **your** box vs someone else's box.